css/global.css

```diff
@@ -2,3 +2,7 @@
 div.feed-shared-update-v2:has(div):has(div.update-components-actor):has(.app-aware-link):has(div.update-components-actor__meta):has(span.update-components-actor__sub-description):has(span.visually-hidden):has(span[aria-hidden="true"]):has(span):not(:has([type="globe-americas"])){
```

Comment:
I kept this because I think they're probably phasing out the old ads to replace them with the new - possibly because of this extension and others like it. But I haven't tested yet to see if the old ads are still there, so in the future I may remove this.